gadgets/fsnotify/artifacthub-pkg.yml

```diff
@@ -0,0 +1,29 @@
+# Artifact Hub package metadata file
+version: 0.31.0
+name: "fsnotify"
+category: monitoring-logging
+displayName: "fsnotify"
+createdAt: "2024-08-12T14:51:00Z"
+digest: "2024-08-12T14:51:00Z"
+description: "enrich inotify or fanotify events"
+logoURL: "https://inspektor-gadget.io/media/brand-icon.svg"
+license: ""
+homeURL: "https://inspektor-gadget.io/docs/latest/gadgets"
```

Comment:
I removed the /fsnotify suffix because the link is not yet valid.
How to resolve this chicken egg problem?

Reply:
I think it's better to use the right link. We should have an automatic way to update the website when changes are done to the docs folder here. I'll think about it.

Reply:
For reference, it was done in #3342.